athletics.lic

```diff
@@ -0,0 +1,170 @@
+custom_require.call(%w(common common-travel))
+
+class Athletics
+  
+  include DRC
+  include DRCT
+  
+  def initialize
+    @settings = get_settings()
+    @athletics_options = get_data('athletics').athletics_options
+
+    @climbing_target = get_data('athletics').practice_options[@settings.climbing_target]
+    
+    @start_skill=DRSkill.getxp('Athletics')
+    @end_skill=@start_skill+15
+    if(@end_skill>29)
+      @end_skill=29
+    end
+
+    if @climbing_target
```

Comment:
I'm not sure if a `case`/`when` structure would make this clearer, have you considered that?